pkgs/os-specific/linux/sssd/default.nix

```diff
@@ -72,8 +72,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-JN4GVx5rBfNBLaMpLcKgyd+CyNDafz85BXUcfg5kDXQ=";
   };
 
+  patches = [ ./fix-ldb-modules-path.patch ];
```

Comment:
```suggestion
  patches = [
    (replaceVars ./fix-ldb-modules-path.patch {
      inherit ldb;
      out = placeholder "out";
    })
  ];
```

Reply:
Hi,

That the first approach have try.
But this fail tests.

Best Regards

Reply:
Hi, 

My only idea was to ask @wolfgangwalther if new method is available ?
Because @wolfgangwalther have exaplain the issue with that : 

> Using replaceVars with placeholder "..." will never work as expected. The placeholder will refer to the derivation created by replaceVars, not to the derivation that this is called from. This was the same with substituteAll previously.

Best Regards